common/src/lib.rs

```diff
@@ -82,10 +83,11 @@ pub fn create_default_consumer(
         .create()
         .expect("kafka consumer should be created");
```

Comment:
Should probably just remove this `except` as well, it is non-intuitive to have multiple error handling schemes used in the same function.